softwareconfiguration/mlplan/weka/src/main/java/ai/libs/mlplan/multiclass/wekamlplan/weka/MLPipelineComponentInstanceFactory.java

```diff
@@ -36,9 +35,7 @@
 	 * @param components
 	 * @throws IOException
```

Comment:
Can we remove the @throws annotation now?

Reply:
yes. How did you notice this? Do you have tool support for this?

Reply:
Let's just say I worked in a company where I learned to see these things as my code was rejected in reviews otherwise :D.